packages/cursorless-engine/src/languages/TreeSitterQuery/validateQueryCaptures.test.ts

```diff
@@ -81,11 +81,6 @@ const testCases: { name: string; isOk: boolean; content: string }[] = [
     isOk: false,
     content: "(if_statement) @statement.leading.start",
   },
-  {
-    name: "Text fragment removal",
-    isOk: false,
-    content: "(comment) @textFragment.removal",
-  },
```

Comment:
Why do we want to support text fragment removal? Guessing you were forced into this one, but seems like a bit of a smell if so

Reply:
Yeah now a text fragment is just like any scope so potentially you could add a removal range for it even if it's not used. I don't know if I think that is a problem or not by itself?

Reply:
I guess that's fine. You can prob remove the special casing code for text fragments in the validator now too I believe https://github.com/cursorless-dev/cursorless/blob/820732a3dbc449508b0a6ae00998cf8d4630f8f4/packages/cursorless-engine/src/languages/TreeSitterQuery/validateQueryCaptures.ts#L5

Reply:
done!